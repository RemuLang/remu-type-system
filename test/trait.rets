store 0 = '0;
store 1 = forall {a} {^1 '0} => '0 -> a;
store 2 = '2;
'2 = '1;
^3 -> ^2 = '2;
