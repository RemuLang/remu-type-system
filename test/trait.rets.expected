0 ^3;
1 forall {a} {^1 ^3} => ^3 -> a;
2 {^1 ^3} => ^3 -> ^2;
